http_request_translator/base.py

```diff
@@ -186,3 +193,22 @@ def encode_url(self, url):
         if self.details.get('data') and (self.details.get('method', '').lower() in http_verb_with_encoding):
             encoded_url += quote(self.details['data'], '')
         return encoded_url
+
+    @staticmethod
+    def load_attributes(cls):
+        """Loads attributes to Script class from a given script's template
+
+        Imports the template file/module, assigns all the attributes defined in the template file to the given class.
+
+        :param class cls: Script class to which template is to be loaded.
+        :param str template: Name of the corresponding template file kept in the default template directory.
+        """
+        if not hasattr(cls, '__language__'):
+            raise ValueError("__language__ not found in class: {}, attributes cannot be loaded".format(cls.__name__))
+
+        template = import_module("{templates_path}.{class_template}".format(templates_path=cls._templates_path,
+                                                                            class_template=cls.__language__))
+        attributes = [x for x in dir(template) if not x.startswith('__')]
```

Comment:
1. Do not use `x` for variable name. It doesn't help understand the code.
2. Use `()` instead of `[]` to use iterators and avoid wasting memory
3. Are we sure that we want all attributes? Shouldn't we use `template.vars()` instead and filter variables starting with `code_`?